src/main/java/com/nitnelave/CreeperHeal/block/CreeperBrick.java

```diff
@@ -8,12 +9,12 @@
 class CreeperBrick extends CreeperBlock
 {
 
-    protected CreeperBrick(BlockState blockState)
+    CreeperBrick(BlockState blockState)
     {
         super(blockState);
 
-        if (CreeperConfig.getBool(CfgVal.CRACK_DESTROYED_BRICKS) && getRawData() == (byte) 0)
-            blockState.setRawData((byte) 2);
+        if (CreeperConfig.getBool(CfgVal.CRACK_DESTROYED_BRICKS))
```

Comment:
I'm not sure this does what we want. There are several types of bricks: mossy, chiseled, plain and cracked. We only want to transform a plain into a cracked.